src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -2140,6 +2144,85 @@ extern "C" {
 
     pub fn mimmutable(addr: *mut ::c_void, len: ::size_t) -> ::c_int;
 
+    // Added in `OpenBSD 5.2`
```

Comment:
I am unsure that the comment with the history add lot of value

Reply:
I was just copying what I saw earlier, but I agree